app.rb

```diff
@@ -0,0 +1,53 @@
+require 'sinatra/base'
+require './lib/user'
+require 'sinatra/flash'
+
```

Comment:
WOW! Impressed you have made this an actual app with databases and user sessions etc!!!